cmd/ncdu/ncdu.go

```diff
@@ -105,6 +105,7 @@ type UI struct {
 	listing        bool          // whether listing is in progress
 	showGraph      bool          // toggle showing graph
 	showCounts     bool          // toggle showing counts
+	showEmptyDir   bool          // toggle showing emptyDir
```

Comment:
This is only used in the Draw function so can be local to the `Draw` function.

Reply:
You're right, I fixed it.